regression/CMakeLists.txt

```diff
@@ -40,13 +40,13 @@ endif()
 list(APPEND GSI_REG_TEST_NAMES
   global_4denvar
   rtma
-  rrfs_3denvar_glbens netcdf_fv3_regional
+  rrfs_3denvar_rdasens
   hafs_4denvar_glbens hafs_3denvar_hybens
 )
 
 # EnKF regression test names
 list(APPEND ENKF_REG_TEST_NAMES
-  global_enkf
+  global_enkf rrfs_enkf_conv
```

Comment:
@hu5970  as you mentioned,  the regional enkf was compiled separately ,and, so, are you still trying to implement ctest for regional enkf to deal with a very complicated "workflow"?

Reply:
I have comment out the regional EnKF case in multi_regression.sh so we do not need to deal with a complicated "workflow". But I would like to commit the regional EnKF case. The next step is to check EnKF code to see if we can merge the regional and global code to make a single version EnKF executable for both global and regional enkf runs.

Reply:
@hu5970  I am wondering how long it will take to have a merged version of enkf  if it is finally decided to be done. 
However, it is  ok for me  to put all those data/configure files for regional enkf regression test data in.  Just wondering if  it could be considered  to change GSI cmake, when regional enkf is to be built,  let the gsi cmake create the regional enkf test rather than, won't create any ctests at all as currently setup ?   Or  create an separate script to allow users to make use of those data to run that regression test for regional enkf after running the gsi building with regonal enkf option?